nucypher/characters/lawful.py

```diff
@@ -1550,11 +1550,9 @@ def __init__(self, policy_encrypting_key=None, controller: bool = True, *args, *
         self.log = Logger(f'{self.__class__.__name__}-{bytes(self.public_keys(SigningPower)).hex()[:6]}')
         self.log.info(self.banner.format(policy_encrypting_key))
 
-    def encrypt_message(self,
-                        message: bytes
-                        ) -> Tuple[UmbralMessageKit, Signature]:
+    def encrypt_message(self, plaintext: bytes) -> Tuple[UmbralMessageKit, Signature]:
```

Comment:
Maybe this method should be renamed to simply `encrypt`. Not necessarily in this PR, but I think it makes sense.

Reply:
#2107